src/core.ts

```diff
@@ -589,12 +601,36 @@ export function stressSolution(path: string, times: number) {
     let solCompileResult = compileCode(sol, out);
     if (solCompileResult.status !== 0) {
         vscode.window.showErrorMessage(`Compilation Error. ${sol}`);
+        let error_path = join(path, "stderr");
+        let error_file = openSync(error_path, "w");
+        writeSync(error_file, solCompileResult.stderr.toString());
+        vscode.commands.executeCommand("vscode.setEditorLayout", {
+            orientation: 1,
+            groups: [{ groups: [{}, {}], size: 0.5 }],
+        });
+        vscode.commands.executeCommand(
+            "vscode.open",
+            vscode.Uri.file(error_path),
+            vscode.ViewColumn.Two
+        );
```

Comment:
What do you think about creating a function with this logic, since it is repeated across the code several times.

It can be something like:

```
function showCompileError(path, result) 
{
     ...
}
```